panic: unaligned 64-bit atomic operation on AWS Fargate X86_64 CPU architecture

##### Versions

| Sarama | Kafka | Go |
|--------|-------|----|
| 1.36.0 | 3.2.0 | 1.18  |

##### Configuration

I'm using default configurations `sarama.NewConfig()`

##### Problem Description

Deploying my consumer to AWS Fargate on X86_64 CPU architecture, causes panic:
```
panic: unaligned 64-bit atomic operation
goroutine 1 [running]:

runtime/internal/atomic.panicUnaligned()
/usr/local/go/src/runtime/internal/atomic/unaligned.go:8 +0x2d	

runtime/internal/atomic.Load64(0xb6a8b34)
/usr/local/go/src/runtime/internal/atomic/atomic_386.s:225 +0x10

github.com/Shopify/sarama.(*client).tryRefreshMetadata(0xb6a8ae0, {0x0, 0x0, 0x0}, 0x3, {0x0, 0x0, 0x0})
/go/pkg/mod/github.com/!shopify/sarama@v1.36.0/client.go:916 +0x43f

github.com/Shopify/sarama.(*client).RefreshMetadata(0xb6a8ae0, {0x0, 0x0, 0x0})
/go/pkg/mod/github.com/!shopify/sarama@v1.36.0/client.go:499 +0x13f

github.com/Shopify/sarama.NewClient({0xb7005e8, 0x1, 0x1}, 0xb57e6c0)
/go/pkg/mod/github.com/!shopify/sarama@v1.36.0/client.go:174 +0x22b

github.com/Shopify/sarama.NewConsumerGroup({0xb7005e8, 0x1, 0x1}, {0x95f447b, 0x7}, 0xb57e6c0)
/go/pkg/mod/github.com/!shopify/sarama@v1.36.0/consumer_group.go:98 +0x3d
```

According to this issue https://github.com/golang/go/issues/41970 the issue might be fixed by changing the order of the field `updateMetaDataMs int64` in the `client` struct defined in the `client.go`

--- a/client.go
+++ b/client.go
@@ -128,6 +128,11 @@ const (
 )
 
 type client struct {
+	// updateMetaDataMs stores the time at which metadata was lasted updated.
+	// Note: this accessed atomically so must be the first word in the struct
+	// as per golang/go#41970
+	updateMetaDataMs int64
+
 	conf           *Config
 	closer, closed chan none // for shutting down background metadata updater
 
@@ -150,7 +155,6 @@ type client struct {
 
 	lock sync.RWMutex // protects access to the maps that hold cluster state.
 
-	updateMetaDataMs int64 // store update metadata time
 }
 
 // NewClient creates a new Client. It connects to one of the given broker addresses

